android/src/main/java/com/theoplayer/ReactTHEOplayerView.kt

```diff
@@ -53,23 +53,22 @@ class ReactTHEOplayerView(private val reactContext: ThemedReactContext) :
     playerContext = ReactTHEOplayerContext.create(
       reactContext,
       PlayerConfigAdapter(configProps)
-    )
-    playerContext?.let {
+    ).apply {
```

Comment:
`apply` doesn't create an additional hidden var and use context `this` instead of  `it` so I'd suggest to use apply here

Reply:
this one is minor change and doesn't relate to fix itself